### ISSUE TITLE
MVC Template with Database Users Template confuses email and username.

### ISSUE THREAD

**Issue by shawnwildermuth** [aspnet/Scaffolding#600]

**Comment by shawnwildermuth** [comment]
Any word on this?

**Comment by prafullbhosale** [comment]
This looks like a template issue. cc @mlorbetske @seancpeters

**Comment by seancpeters** [comment]
cc @danroth27 
From a quick search, this appears to be the case for all versions of both StarterWeb C# & Razor templates.

**Comment by shawnwildermuth** [comment]
Yup.

**Comment by danroth27** [comment]
Is the proposal here to simply rename the LoginViewModel.Email property to LoginViewModel.Username?

**Comment by shawnwildermuth** [comment]
That would fix it, or to change the generated HTML to say username. Probably need both.

**Comment by danroth27** [comment]
Well, the template is written with the assumption that the username will be an email address. If we change the HTML to say username, then it will no longer be clear to the user that an email address is really what is expected.

**Comment by shawnwildermuth** [comment]
I tend to agree. That's the confusing part. But if you change the model, it should be sufficient. But I lost about two hours figuring it out when I *didn't* use email as my username. There are two fields that could have an email and  assuming that email is the username, why have both?

**Comment by vijayrkn** [comment]
The templates needs to be updated first before updating the scaffolding code. This issue should be moved to AspNetCore repo 

https://github.com/aspnet/AspNetCore/blob/d4f7a199c7616a6e2c3374c33587d65d83882904/src/Identity/UI/src/Areas/Identity/Pages/V3/Account/ResendEmailConfirmation.cshtml.cs#L43
